keyboards/synthandkeys/bento_box/info.json

```
@@ -0,0 +1,66 @@
+{
+    "manufacturer": "Synth-and-Keys",
+    "keyboard_name": "bento_box",
+    "maintainer": "Synth-and-Keys",
+    "bootloader": "stm32-dfu",
+    "bootmagic": {
+        "matrix": [2, 0]
+    },
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "B2","pin_b": "B1"},
+            {"pin_a": "A13","pin_b": "A14"},
+            {"pin_a": "A15","pin_b": "B3"}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "direct": [
+            ["A9", "A8"],
+            ["B9", "B6"],
+            ["B0", "A5", "A2"],
+            ["A7", "A4", "A1", "B8", "B5"],
+            ["A6", "A3", "A0", "B7", "B4"]
```

You can make better use of the matrix like so:
```suggestion
            ["A9", "A8", "B9", "B6", "B0", "A5"],
            ["A2", "A7", "A4", "A1", "B8", "B5"],
            ["A6", "A3", "A0", "B7", "B4", null]
```
Then adjust the layout matrix positions appropriately.